internal/workos/workos.go

```diff
@@ -2,5 +2,5 @@ package workos
 
 const (
 	// Version represents the SDK version number.
-	Version = "v1.16.0"
+	Version = "v1.17.0"
```

Comment:
Could we omit the version change from the PR?

We bump the version separately when we're ready to cut a new SDK release.

Reply:
Done